pango/src/glyph_string.rs

```diff
@@ -32,4 +32,63 @@ impl GlyphString {
             }
         }
     }
+
+    pub fn log_clusters(&self) -> &[i32] {
+        unsafe {
+            let ptr = (*self.to_glib_none().0).log_clusters as *const i32;
```

Comment:
https://gtk-rs.org/gtk-rs-core/git/docs/glib/collections/struct.Slice.html#method.from_glib_borrow_num if you feel like it :)

Reply:
Fixed, and added `Slice::from_glib_borrow_num_mut` too